controllers/programmes/index.js

```diff
@@ -134,4 +136,16 @@ router.get('/:slug', injectFundingProgramme, (req, res, next) => {
     }
 });
 
+router.use(
```

Comment:
Thinking about it it might be better to do something like:

```suggestion
router.use(
    '/building-better-opportunities/*',
    (req, res, next) => {
            if (customAncestors) {
                res.locals.customAncestors = customAncestors;
            }
            next();
        },
    basicContent()
);
```

And then we could remove the middleware from `basicContent`

Reply:
Good call, have a look now